src/main/java/org/codehaus/plexus/archiver/BaseFileSet.java

```diff
@@ -64,5 +66,12 @@
      * @return The transformers.
      */
     InputStreamTransformer getStreamTransformer();
+    
+    /**
+     * Returns a set of file mappers, which should be used
```

Comment:
I think it would be better if the comment explicitly states that only the file names are changed. Otherwise combined with the bit misleading name of `FileMapper` somebody may get the false impression that the files themselves are changed. Of course a quick look at the `FileMapper` docs would make that clear but I think it will be more developer friendly if the comment is explicit.

Reply:
of course you right, I add more explicit comment